pkgs/applications/science/logic/boolector/default.nix

```
@@ -13,6 +13,14 @@ stdenv.mkDerivation rec {
     sha256 = "1smcy6yp8wvnw2brgnv5bf40v87k4v4fbdbrhi7987vja632k50z";
   };
 
+  patches = [
+    (fetchpatch {
```

can we add a comment as to when this should be removed
```suggestion
    # merged into master, remove after 3.2.2
    (fetchpatch {
```